controllers/admin/AdminStatsController.php

```diff
@@ -684,7 +689,13 @@ public function displayAjaxGetKpi()
                 break;
 
             case 'frontoffice_translations':
-                $themes = Theme::getThemes();
+                $theme_manager = new ThemeManager(
```

Comment:
Third time it is newed, starting to think a factory / builder really makes sense!